apis/v1alpha1/rhmi_types.go

```diff
@@ -210,6 +210,11 @@ type AlertingEmailAddresses struct {
 	CSSRE        string `json:"cssre"`
 }
 
+type CustomSmtpStatus struct {
+	Active bool   `json:"active"`
```

Comment:
Small nit -
we use `Enabled bool` for SMTPEnabled in the cr. 
Could we use Enabled here as well instead of Active?